docs/git-compatibility.md

```diff
@@ -31,10 +31,13 @@ a comparison with Git, including how workflows are different, see the
   and [how they interoperate with Git](#branches).
 * **Tags: Partial.** You can check out tagged commits by name (pointed to be
   either annotated or lightweight tags), but you cannot create new tags.
-* **.gitignore: Yes.** Ignores in `.gitignore` files are supported. So are
+* **.gitignore: Yes.** Patterns in `.gitignore` files are supported. So are
   ignores in `.git/info/exclude` or configured via Git's `core.excludesfile`
-  config. The `.gitignore` support uses a native implementation, so please
-  report a bug if you notice any difference compared to `git`.
+  config. Since working-copy files are snapshotted by every `jj` command, you
+  might need to run `jj file untrack` to exclude newly ignored files from the
+  working-copy commit. It's recommended to set up the ignore patterns earlier.
```

Comment:
Shouldn't that say "It is required" instead of "recommended"? If not, the user actually gets an error: 
```
$ jj file untrack README.md 
Error: 'README.md' is not ignored.
Hint: Files that are not ignored will be added back by the next command.
Make sure they're ignored, then try again.
```

Reply:
What I meant was to add `.gitignore` (or global ignore) before executing `jj status`, etc.
https://github.com/martinvonz/jj/issues/5059#issuecomment-2536071114

Better to remove the sentence if it's unclear?